tests/test_core.py

```diff
@@ -4666,6 +4666,11 @@ def add_pre_run(self, code):
       f.write('Module.preRun = function() { %s }' % code)
     self.emcc_args = ['--pre-js', 'pre.js']
 
+  def add_post_run(self, code):
+    with open('post.js', 'w') as f:
+      f.write('Module.postRun = function() { %s }' % code)
+    self.emcc_args = ['--post-js', 'post.js']
```

Comment:
Surely `+=` here?

Reply:
Should this be a pre-js? If we're not running synchronously, this will get called after we instantiate.